lib/main.dart

```diff
@@ -45,14 +45,14 @@ class MyHomeRoute extends StatefulWidget {
 
 
 class _MyHomeRouteState extends State<MyHomeRoute>{
-  String username = "";
```

Comment:
Strings are usually represented by double quotes
And characters with single quotes

Reply:
I was following the style guide's instructions, just as they suggest - https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#prefer-single-quotes-for-strings